cores/arduino/HardwareTimer.cpp

```diff
@@ -898,6 +898,11 @@ timer_index_t get_timer_index(TIM_TypeDef *instance)
   */
 uint32_t HardwareTimer::getTimerClkFreq()
 {
+#if defined(STM32MP1xx)
+  uint8_t timerClkSrc = getTimerClkSrc(_HardwareTimerObj.handle.Instance);
```

Comment:
Renamed in the master

```suggestion
  uint8_t timerClkSrc = getTimerClkSrc(_timerObj.handle.Instance);
```